frontend/app/src/theme/community/solarizeddark.ts

```diff
@@ -0,0 +1,66 @@
+import type { Theme } from "../types";
+
+const veryDarkCyan = "#002b36";
+const darkCyan = "#073642";
+const darkCyan2 = "#0c586ba6";
+const darkBlue = "#186899";
+const gray = "#8b9898";
+const lightishGray = "#90aba7";
+const lightGray = "#d6dddc";
+const olive = "#748c09";
+
+const txt = lightGray;
+const txtLight = lightishGray;
+
+export function getTheme(base: Theme): Theme {
+    base.name = "solarizeddark";
+    base.label = "Solarized Dark";
+    base.burst = false;
+    base.bg = veryDarkCyan;
+    base.bd = gray;

```

Comment:
I think the border is a tiny bit in your face and personally prefer something like 

```
base.bd = "rgba(255,255,255,0.2)";
```